.github/workflows/ansible-lint-sap_storage_setup.yml

```diff
@@ -0,0 +1,43 @@
+---
+
+# Workflow for ansible-lint of a role
+
+name: ansible-lint of the role sap_storage_setup
+
+on:
+  push:
+    branches:
+      - main
+      - dev
+    paths:
+      - 'roles/sap_storage_setup/**'
+  pull_request:
+    branches:
+      - main
+      - dev
+    paths:
+      - 'roles/sap_storage_setup/**'
+
+jobs:
+  ansible-lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out the code
+        uses: actions/checkout@main
```

Comment:
```suggestion
        uses: actions/checkout@v3
```

Reply:
Closing for now, see https://github.com/sap-linuxlab/community.sap_install/pull/419#discussion_r1270607762.